README.md

```diff
@@ -2,6 +2,84 @@
 
 This action runs either spelling, url, or quiz format checks in an [OTTR course](https://github.com/jhudsl/OTTR_Template).
 
+## main.yml
+
+This GitHub Actions (GHA) workflow, "Testing ottr-reports", performs a series of checks on pull requests made to the main branch and can also be manually triggered using the workflow_dispatch event.
+
+It is composed of four jobs: `get_branch`, `spell-check`, `urls-check`, and `quiz-format`:
+
+1. `get_branch` Job
+
+Purpose: Determines the branch name that triggered the workflow and makes it available for subsequent jobs.
+
+Steps:
```

Comment:
```suggestion
**Steps:**
```
Some minor formatting things like this (while not completely necessary) are nice to make it easy for eyes to follow.